packages/mco/components/mco-dashboard/data-policy/cluster-app-card/cluster.tsx

```diff
@@ -145,35 +149,11 @@ export const HealthSection: React.FC<HealthSectionProps> = ({
 };
 
 export const PeerConnectionSection: React.FC<PeerConnectionSectionProps> = ({
-  clusterName,
+  peerClusters,
 }) => {
   const { t } = useCustomTranslation();
-  const [mirrorPeers, mirrorPeersLoaded, mirrorPeersError] =
-    useK8sWatchResource<MirrorPeerKind[]>({
-      kind: referenceForModel(MirrorPeerModel),
-      isList: true,
-      namespaced: false,
-      cluster: HUB_CLUSTER_NAME,
-    });
-
-  const peerConnectedCount = React.useMemo(() => {
-    if (mirrorPeersLoaded && !mirrorPeersError) {
-      return (
-        mirrorPeers.reduce((acc, mirrorPeer: MirrorPeerKind) => {
-          if (
-            !!mirrorPeer?.spec?.items?.find(
-              (item) => item?.clusterName === clusterName
-            )
-          ) {
-            return acc + 1;
-          }
-          return acc;
-        }, 0) || 0
-      );
-    }
-    return 0;
-  }, [clusterName, mirrorPeers, mirrorPeersLoaded, mirrorPeersError]);
-
+  // Exclude the selected cluster
+  const peerConnectedCount = peerClusters.length - 1;
```

Comment:
```suggestion
  const peerConnectedCount = !!peerClusters.length ?  peerClusters.length - 1 : 0;
```
i am seeing negative value for the peer which dont have any peers